node/cmd/guardiand/node.go

```diff
@@ -295,6 +299,9 @@ func init() {
 	wormchainKeyPath = NodeCmd.Flags().String("wormchainKeyPath", "", "path to wormhole-chain private key for signing transactions")
 	wormchainKeyPassPhrase = NodeCmd.Flags().String("wormchainKeyPassPhrase", "", "pass phrase used to unarmor the wormchain key file")
 
+	ibcContract = NodeCmd.Flags().String("ibcContract", "", "Address of the IBC smart contract on wormchain")
+	ibcConfig = NodeCmd.Flags().String("ibcConfig", "", `List of IBC connections and their associated chain IDs like [{"ChainID":18,"ConnID":"connection-0"},{"ChainID":19,"ConnID":"connection-1"}]`)
+
```

Comment:
Should we add this to DumpRPCs?

Reply:
There is no ibcRPC parameter since it uses the wormchain RPC.